Gemfile

```diff
@@ -1,10 +1,3 @@
 source 'http://rubygems.org'
 
 gemspec
-
-case version = ENV['GRAPE_VERSION'] || '~> 0.14.0'
-when 'HEAD'
-  gem 'grape', github: 'ruby-grape/grape'
-else
-  gem 'grape', version
-end
```

Comment:
I also think we should not spend too much time supporting previous releases of things with newer versions, I suggest moving onward and enabling 0.16.x + whatever new stuff over older things.